src/ui/misc/TabBarPlaceholder.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { View } from 'react-native';
+import { DefaultTabBarHeight, Colors } from '../../styles';
+
+export const TabBarPlaceholder = (props: { color?: string }) => (
+    <View
+        style={{
+            height: DefaultTabBarHeight,
```

Comment:
It seems this is not necessary on android, because there the TabBar is not transparent so the content is not behind it and this just puts a grey bar on the bottom of the screen.